src/components/NavigationBar.jsx

```diff
@@ -249,8 +249,8 @@ const MenuList = ({ isMobile, closeDropdown }) => (
     <MenuItem name="FAQ" href="/#faq" isAnchor isMobile={isMobile} closeDropdown={closeDropdown} />
     <MenuItem name="Sponsors" href="/#sponsors" isAnchor isMobile={isMobile} closeDropdown={closeDropdown} />
     <MenuItem
-      name="2022"
-      href="https://cmd-f2022.nwplus.io"
+      name="2023"
+      href="https://cmd-f.nwplus.io/"
```

Comment:
Could you actually change the target to https://cmd-f2023.nwplus.io/, it'll break for now, but I can update the Cloudflare DNS